manual.md

```diff
@@ -1035,13 +1035,22 @@ Console Print and Input Statements
 **PRINT _expr_ ; ...**  
 
   Outputs strings and numbers to the
-  screen.
+  screen or other output device.
 
   Each _expr_ can be a constant string,
   a string variable or any complex
   expression, with commas or semicolons
   between each expression.
 
+  If the first expression is a device
+  I/O channel (e.g., `PRINT #1;"HELLO"`)
```

Comment:
This is not a good example, because in FastBasic you always use a comma after the I/O channel: `PRINT #1,"HELLO"` :)